src/pydata_sphinx_theme/assets/styles/base/_base.scss

```diff
@@ -177,3 +177,16 @@ pre {
     padding-right: 10px;
   }
 }
+
+// the back to top btn
+#pst-back-to-top {
+  z-index: $zindex-tooltip;
+  position: fixed;
+  display: none;
+  top: calc(var(--pst-header-height) + 1rem);
+  left: 50%;
```

Comment:
Is there a reason why you are using `left: 50%;` vs `left: auto;` is this to avoid clash with other page components?

Reply:
no it's because i'm not expert in css. I assume auto is better ?

Reply:
I just tried what you suggested and the button ended on the far left side of my screen, rolling back to previous version.